src/backend/distributed/planner/multi_explain.c

```diff
@@ -124,85 +118,54 @@ MultiExplainOneQuery(Query *query, IntoClause *into, ExplainState *es,
 	}
 #endif
 
-	/* handle local queries in the same way as ExplainOneQuery */
-	if (localQuery)
-	{
-		PlannedStmt *plan = NULL;
-
-		INSTR_TIME_SET_CURRENT(planStart);
+	/* plan query, just like ExplainOneQuery does */
+	INSTR_TIME_SET_CURRENT(planStart);
 
-		/* plan the query */
-		plan = pg_plan_query(query, cursorOptions, params);
+	/* plan the query */
+	plan = pg_plan_query(query, cursorOptions, params);
 
-		INSTR_TIME_SET_CURRENT(planDuration);
-		INSTR_TIME_SUBTRACT(planDuration, planStart);
+	INSTR_TIME_SET_CURRENT(planDuration);
+	INSTR_TIME_SUBTRACT(planDuration, planStart);
```

Comment:
Oh, I see. The old code pretended to have two paths but just calling the planner is sufficient. It's less that this review is hard to understand and more that I'm having trouble remembering whether there are good reasons for anything here or if it's all cruft. It's seeming like the latter which makes me inclined to want to remove even more.